apps/webcert/src/components/header/WebcertHeader.tsx

```diff
@@ -81,7 +101,7 @@ const WebcertHeader: React.FC<Props> = ({ isEmpty = false }) => {
       banners={[<SystemBanners key="system-banners" />]}
       tabs={tabs}
       onSwitchTab={onSwitchTab}
-      subMenuBanners={missingSubscription() ? [<SystemBanner key="system-banner" banner={subscriptionWarning} />] : []}
```

Comment:
Maybe SystemBanner the component could be moved into AppHeader instead and we just send Banner[] instead? Banner is placed in common so it should be okay